lib/config.py

```diff
@@ -158,16 +158,23 @@ def __init__(self):
         # create the top-level parser
         self.parser = argparse.ArgumentParser()
         self._CONF = None
-        self._setup_config_parser_options()
+        self._setup_config_parser_options(SUBCOMMANDS)
```

Comment:
I'd rewrite the commit message to make the main point clearer. It is not the method rename, it is the addition of the parameter. Also, please try to keep the short commit messages short. It is really annoying to read messages like this:
```
Rename subparsers manager from subparser to subparsers to make code e…

…asier to understand
```
It also only appears partially in the commits page. I suggest:
```
Add "subcommands" parameter to ConfigParser methods

This restricts knowledge of external constants to a single method in the class.
Also, rename the "_add_subparser" method and document related methods.
```